### PR TITLE
Extend configuration for customMetrics

### PR DESCRIPTION
**Pull request by ktibi**
### What does this PR do?

As explained here #196 , the aim is to give you the option of adding more customised metrics configurations.

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [x] Yes, I ran `pre-commit run -a` with this PR
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/examples) to support my PR (when applicable)
- [ ] Yes, I have updated the [Pages](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/docs) for this feature

**Note**: Not all the PRs required examples and docs.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by bonclay7** [comment]
Nice! thanks for this @ktibi! can you install https://pre-commit.com/ and run it on your branch. This will generate terraform docs for the variable changes

**Comment by ktibi** [comment]
@bonclay7 Done !

File: modules/eks-monitoring/variables.tf
Comment (bonclay7) [inline modules/eks-monitoring/variables.tf:1]: I see tests are failing, maybe you need to delete this variable https://github.com/aws-observability/terraform-aws-observability-accelerator/pull/202/files#diff-9cde74449020c39b5f40c238f3746f22685b164126cf5e4c49c052aa931dbc85R233-R244

If possible, can you update the doc with a usage example of the new custom_metrics_config [here](https://github.com/aws-observability/terraform-aws-observability-accelerator/blob/main/docs/eks/index.md?plain=1#L172-L178)

Many thanks again 🙏🏾

File: modules/eks-monitoring/variables.tf [hunk 1]
```diff
@@ -245,6 +245,20 @@ variable "custom_metrics_config" {
   }
 }
 
+variable "custom_metrics_config" {
+  description = "Configuration object to custom metrics collection"
```
Comment (bonclay7) [inline modules/eks-monitoring/variables.tf:249]: I think you forgot `enable`

Comment (bonclay7) [reply modules/eks-monitoring/variables.tf:234]: Yes, you're right, I was mentioning just a typo in the description 
`"Configuration object to custom metrics collection"`

**Comment by ktibi** [comment]
@bonclay7 Do you know why tests fail ?

**Review by bonclay7 — APPROVED** [review]
Thanks again!

**Comment by bonclay7** [comment]
Bundled in the latest release https://github.com/aws-observability/terraform-aws-observability-accelerator/releases/tag/v2.5.5

**Pull request merged**